src/BloomExe/Edit/PageListView.cs

```diff
@@ -32,24 +33,43 @@ public partial class PageListView : UserControl
 			_thumbNailList.PageSelectedChanged+=new EventHandler(OnPageSelectedChanged);
 			_thumbNailList.Isolator = isolator;
 			_thumbNailList.ControlKeyEvent = controlKeyEvent;
+			// First action determines whether the menu item is enabled, second performs it.
+			var menuItems = new List<Tuple<string, Func<IPage, bool>, Action<IPage>>>();
+			menuItems.Add(
+				Tuple.Create<string,  Func<IPage, bool>, Action<IPage>>(
+					LocalizationManager.GetString("EditTab._duplicatePageButton", "Duplicate Page"), // same ID as button in toolbar));
+					(page) => page != null && !page.Required && !_model.CurrentBook.LockedDown,
+					(page) => _model.DuplicatePage(page)));
+			menuItems.Add(
+				Tuple.Create<string,  Func<IPage, bool>, Action<IPage>>(
+					LocalizationManager.GetString("EditTab._deletePageButton", "Remove Page"),  // same ID as button in toolbar));
+					(page) => page != null && !page.Required && !_model.CurrentBook.LockedDown,
+					(page) =>
+					{
+						if (ConfirmRemovePageDialog.Confirm())
+							_model.DeletePage(page);
+					}));
+			menuItems.Add(
+				Tuple.Create<string,  Func<IPage, bool>, Action<IPage>>(
+					LocalizationManager.GetString("EditTab._chooseLayoutButton", "Choose Different Layout"),
```

Comment:
While I know you're just being consistent, localization Ids really don't want to start with underscores.